lms/envs/common.py

```diff
@@ -1205,6 +1205,10 @@ def _make_mako_template_dirs(settings):
 
 XBLOCK_FS_STORAGE_BUCKET = None
 XBLOCK_FS_STORAGE_PREFIX = None
+
+# .. setting_name: XBLOCK_SETTINGS
+# .. setting_default: {}
+# .. setting_description: Dictionary containing server-wide configuration of XBlocks on a per-type basis
```

Comment:
I think this could use a little more context from https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/services.py#L15-L23.  Additionally, you could state that additional details are available in the ServerSettings documentation, and potentially even provide a link to master without a line number, since it could become outdated.

Thoughts?

Reply:
1. yeah I'll add the explanation on the key names
2. I'll include the `SettingsService` path for more reference

Reply:
@robrap I added the recommendations, please let me know what you think